pkg/apis/vpc/v1alpha1/subnetset_types.go

```diff
@@ -35,6 +35,9 @@ type SubnetSetSpec struct {
 
 	// Subnet DHCP configuration.
 	SubnetDHCPConfig SubnetDHCPConfig `json:"subnetDHCPConfig,omitempty"`
+	// Pre-created Subnets names owned by this SubnetSet. It's exclusive with other fields like IPv4SubnetSize, AccessMode, and SubnetDHCPConfig.
+	// Once this field is set, the other fields are ignored.
```

Comment:
Just to confirm, are other fields being ignored or they cannot be set, i.e. rejected by validation rule or webhook?

Reply:
rejected by webhook

Reply:
Then shall we mention `Once this field is set, the other fields cannot be set` instead of saying they are ignored?

Reply:
Updated.